src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -55,17 +77,42 @@ public function register_routes() {
 	 * @since 5.0.0
 	 *
 	 * @param WP_REST_Request $request Full details about the request.
-	 * @return true|WP_Error True if the request has read access for the item, otherwise WP_Error object.
+	 * @return bool|WP_Error True if the request has read access for the item, otherwise WP_Error object.
```

Comment:
Let's keep this as `true|WP_Error` and return a descriptive error in `check_read_active_theme_permission`.